src/modules/market/components/market-portfolio-card/market-portfolio-card.jsx

```diff
@@ -139,7 +139,12 @@ export default class MarketPortfolioCard extends Component {
         </section>
         <section className={Styles.MarketCard__tablesection}>
           {(myPositionOutcomes || []).filter(outcome => outcome.position).length !== 0 &&
-            <div className={Styles.MarketCard__headingcontainer}>
+            <div
+              className={Styles.MarketCard__headingcontainer}
+              role="button"
+              tabIndex="0"
```

Comment:
The tabIndex causes a glowing blue halo to appear on click. Moving the tabindex to the svg button fixes the problem. If we are supporting tabbing we need to add a focus indicator to the svg button.

<img width="1469" alt="screen shot 2018-02-26 at 4 40 13 pm" src="https://user-images.githubusercontent.com/328965/36704121-c51f3c76-1b13-11e8-8cc2-6fc51084937d.png">

Reply:
can't satisfy the linter and this requirement, i can just remove the halo by removing it in CSS, but it has to be tabbable if it's going to have an onclick handler for accessibility reasons